api/cli/defaults.go

```diff
@@ -13,5 +13,8 @@ const (
 	defaultTLSCACertPath   = "/certs/ca.pem"
 	defaultTLSCertPath     = "/certs/cert.pem"
 	defaultTLSKeyPath      = "/certs/key.pem"
+	defaultSSL             = "false"
```

Comment:
Please add defaults flags for Windows in the `api/cli/defaults_windows.go` file.